crates/rapier_testbed2d-f64/Cargo.toml

```diff
@@ -34,12 +34,12 @@ default = ["dim2"]
 dim2 = []
 parallel = ["rapier/parallel", "num_cpus"]
 other-backends = ["wrapped2d"]
-profiling = ["dep:puffin_egui", "profiling/profile-with-puffin"]
+profiler_ui = ["dep:puffin_egui", "profiling/profile-with-puffin"]
 # See https://github.com/dimforge/rapier/issues/760.
 unstable-puffin-pr-235 = []
 
 [package.metadata.docs.rs]
-features = ["parallel", "profiling"]
+features = ["parallel", "profiler_ui"]
```

Comment:
this feature change name is to avoid naming a feature the same as a dependency, + better represent what this does.

Reply:
I'm not entirely sure, but I encountered problems with enabling `profiling/profile-with-tracy` without that change (because you cannot profile with 2 backends (puffin+tracy)), I imagine that enabled `profiling` too, so that feature name change would have fixed it.